app/templates/admin/messages.hbs

```diff
@@ -1 +1,58 @@
-{{outlet}}
+<div class="ui grid stackable">
+  <div class="row">
+    <div class="sixteen wide column">
+      <table class="ui stackable very basic table compact">
+        <thead>
+          <tr>
+            <th>{{t 'Recipient'}}</th>
+            <th>{{t 'Trigger'}}</th>
+            <th>{{t 'Email Message'}}</th>
+            <th>{{t 'Notification Message'}}</th>
+            <th>{{t 'Options'}}</th>
+            <th>{{t 'Sent At'}}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each model as |message|}}
+            <tr>
+              <td>
+                <div class="ui ordered list">
+                  {{#each message.recipient as |recipient|}}
+                    <div class="item">{{recipient.name}}</div>
+                  {{/each}}
+                </div>
+              </td>
+              <td>
+                {{message.trigger}}
+              </td>
+              <td>
+                <h5 class="ui header">{{message.emailMessage.subject}}</h5>
+                {{message.emailMessage.message}}
+              </td>
+              <td>
+                <h5 class="ui header">{{message.notificationMessage.subject}}</h5>
+                {{message.notificationMessage.message}}
+              </td>
+              <td>
+                <div class="ui ordered list">
```

Comment:
@utkarshnath Actually this is a checkbox list. It was missing from the screenshot attached to the issue please check the screenshot:
![screenshot from 2017-06-15 13-08-36](https://user-images.githubusercontent.com/12807846/27170503-c5a02a5a-51cb-11e7-9356-62cf854ea97e.png)